src/js/libgeo/GeoOps.js

```diff
@@ -2426,6 +2441,9 @@ geoOps.TrReflectionP.updatePosition = function(el) {
 geoOps.TrReflectionL = {};
 geoOps.TrReflectionL.kind = "Tr";
 geoOps.TrReflectionL.signature = ["L"];
+geoOps.TrReflectionL.initialize = function(el) {
+    el.isEuclidean = -1;
```

Comment:
True. The problem this abuse of the field is trying to achieve is that we have both orientation-preserving and orientation-reversing operations. And while it's easy enough to distinguish these on real coordinates, that distinction is far harder along complex detours. On the other hand, the distinction is evident in the construction sequence, just as the classification as a similarity is evident there. So that's why I'm abusing the flag in this way. `similarityDirection` or `similaritySign` or something like this might be alternatives, although they would be more confusing in contexts where the sign does not matter and the boolean coercion is really all that matters. Do you or someone else have any better names to suggest?